components/product-page/products.css

```diff
@@ -0,0 +1,60 @@
+/* ------------------------|| IMPORTS -----------------------*/
+@import url("../../utilities/main.css");
+
+/* ---------------------- || PRODUCTS PAGE --------------------------- */
+.products-main {
+  display: flex;
+}
+
+/* ------------------------ || PRODUCTS SIDEBAR --------------------------------*/
+.products-sidebar {
+  flex-basis: 28rem;
+}
+.sidebar-list {
+  list-style: none;
+  padding: 1.5rem;
+}
+.sidebar-list-item {
+  margin-bottom: 1.8rem;
+}
+.list-filter-clear {
+  float: right;
+  text-decoration: underline;
+}
+.list-filter-header {
+  font-weight: 800;
+}
+
+/* ---------------------- || SIDEBAR LIST --------------------------------*/
+.item-label {
+  display: inline-block;
+  margin-top: 1rem;
+}
+.item-input {
+  margin-right: 0.3rem;
+}
+
+/* ---------------------- || PRODUCTS SECTION --------------------------------*/
+.products-section {
+  background-color: #f5f5f5;
```

Comment:
use color variable

Reply:
ya that i will do once i finish this site.